src/core_modules/capture-core/components/WidgetsChangelog/Changelog/Changelog.container.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import React from 'react';
+import { Modal } from '@dhis2/ui';
+import { useChangelogData } from '../common/hooks';
+import { ChangelogComponent } from './Changelog.component';
+import { CHANGELOG_ENTITY_TYPES } from './index';
+import { LoadingMaskElementCenter } from '../../LoadingMasks';
+import type { ItemDefinitions } from './Changelog.types';
+
+type Props = {
+    entityId: string,
+    entityType: $Values<typeof CHANGELOG_ENTITY_TYPES>,
+    isOpen: boolean,
+    setIsOpen: (boolean | boolean => boolean) => void,
```

Comment:
How about we only have `close: () => void` and handle everything else on the outside?